src/framework/global/runtime.cpp

```diff
@@ -22,11 +22,24 @@
 
 #include "runtime.h"
 
+#ifdef Q_OS_LINUX
+#include <pthread.h>
+#endif
+
 static thread_local std::string s_threadName;
 
 void muse::runtime::setThreadName(const std::string& name)
 {
     s_threadName = name;
+#ifdef Q_OS_LINUX
```

Comment:
As far as I understand, we can enable this code for FreeBSD. Also, pthread_setname_np works on mac too, and it's even easier to use it there:

```
#if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
   // ....
#elif defined(Q_OS_MACOS)
    pthread_setname_np(name.c_str());
#endif
```